### PR TITLE
fix: background script warnings

### PR DESCRIPTION
**Pull request by williamrijksen**
Known issue:
 - [x] Fix `tabs.sendMessage` receiver
 - [x] Fix scribit pro

### CODE REVIEW

File: libs/shared/types/src/a11y.ts [hunk 1]
```diff
@@ -6,9 +6,9 @@ export namespace A11y {
      * List of supported accessibility features.
      */
     export enum Feature {
-        AD = 'audio description',
         CC = 'closed captioning',
         SL = 'sign language',
+        AD = 'audio description',
```
Comment (nstapelbroek) [inline libs/shared/types/src/a11y.ts:11]: Thanks for reordering this one :)

File: libs/feature/browser-extension/src/players/media-element-js.ts [hunk 2]
```diff
@@ -68,7 +68,7 @@ export class MediaElementJS extends FrameworkPlayer<MeJS.Player> implements A11y
      */
     private adjustFeature(feature: string, enabled: boolean): void {
         for (let player of this.players) {
-            const position = this.getFeaturePosition(player, feature);
+            const position = MediaElementJS.getFeaturePosition(player, feature);
```
Comment (nstapelbroek) [inline libs/feature/browser-extension/src/players/media-element-js.ts:71]: Why did this change?

File: libs/feature/browser-extension/src/media-player-handler.ts [hunk 3]
```diff
@@ -28,10 +27,13 @@ export abstract class MediaPlayerHandler extends A11yHandler {
      * @inheritDoc
      */
     protected transformWatcher(watcher: Observable<MessageEvent>): Observable<MessageEvent> {
-        return watcher.pipe(tap(event => {
-            if (!this.preferences || event.messageType === MessageType.UpdatedTab) {
-                return this.scan();
-            }
-        }));
+        return watcher.pipe(
+            debounceTime(0),
```
Comment (nstapelbroek) [inline libs/feature/browser-extension/src/media-player-handler.ts:31]: This reads as a workaround. Any reason why this is added here?

**Review by nstapelbroek — COMMENTED** [review]
Tested functionally on my chrome, could not  load [the ci artifacts](https://github.com/Accessibility-Video/accessibility-plugin/actions/runs/1650124269) in firefox. The rest looks good and I merely left some comments in the PR.

File: libs/feature/browser-extension/src/players/media-element-js.ts (hunk 2 above)
Comment (williamrijksen) [reply libs/feature/browser-extension/src/players/media-element-js.ts:71]: Was suggested by my IDE, didn't had a proper look at it tbh

**Pull request merged**